_projects/13-build-tools.md

```diff
@@ -8,9 +8,23 @@ status: Active
 type: project
 active: true
 home: false
-description: "Performance improvements, new features and simplifications in sbt"
+description: "Contributions to Zinc, Scala's incremental compiler, and sbt."
 ---
 
-[Zinc Improvements](https://github.com/sbt/zinc/labels/Scala%20Center)
+This initiative focuses on improving day-to-day tools that all Scala developers depend on: [Zinc](https://github.com/sbt/zinc) and [sbt](https://github.com/sbt/sbt).
 
-[Improve builds with direct dependencies](https://github.com/scalacenter/advisoryboard/blob/master/proposals/009-improve-direct-dependency-experience.md)
```

Comment:
You should still include this link somewhere because it's what got this work off the ground...